tests/e2e/e2e_test.go

```diff
@@ -115,17 +115,17 @@ var _ = Describe("odoe2e", func() {
 			Expect(configOutput).To(ContainSubstring("ComponentType"))
 		})
 
-		It("should allow deleting a set config locally", func() {
+		FIt("should allow unsetting a config locally", func() {
```

Comment:
Shouldn't this remain as `It`?

Reply:
Should be ```It```. I guess it is a typo, However surprisingly not caught in CI. It could be a literal in ```GonkGo```.

Reply:
`Fit` is valid - it is Focused Spec - https://onsi.github.io/ginkgo/#focused-specs

Reply:
Ahh...Right
@girishramnani What you want to achieve here. I guess you are using it for your debugging propose, right ?

Reply:
yeah

Reply:
resolved